test/services/notices/setup/fetch-returns-due-by-licence-ref.service.test.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = (exports.lab = Lab.script())
+const { expect } = Code
+
+// Test helpers
+const ReturnLogHelper = require('../../../support/helpers/return-log.helper.js')
+const { generateLicenceRef } = require('../../../support/helpers/licence.helper.js')
+
+// Thing under test
+const FetchReturnsDueByLicenceRefService = require('../../../../app/services/notices/setup/fetch-returns-due-by-licence-ref.service.js')
+
+describe('Notices - Setup - Fetch returns due by licence ref service', () => {
+  let licenceRef
+  let returnLog
+
+  beforeEach(async () => {
+    licenceRef = generateLicenceRef()
+
+    returnLog = await ReturnLogHelper.add({
+      licenceRef,
+      metadata: {
+        purposes: [
+          {
+            tertiary: { description: 'Potable Water Supply - Direct' }
+          }
+        ]
+      }
+    })
+
+    // Add a record not due
+    await ReturnLogHelper.add({
+      licenceRef,
+      status: 'not due'
+    })
```

Comment:
There is no such status as this. To avoid confusion, I would change this to either `completed` or `void`.